### PR TITLE
Add Slovak translation

### PR DESCRIPTION
**Pull request by jose1711**

### CODE REVIEW

File: src/_locales/sk/messages.json [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+        "message": "Prehrať"
+    },
+    "popup_removeItem_title": {
+        "message": "Odstrániť"
+    },
+    "popup_text_placeholder": {
+        "message": "Text na odoslanie"
+    },
+    "popup_done_textcontent": {
+        "message": "Skončiť po odoslaní"
```
Comment (regseb) [inline src/_locales/sk/messages.json:213]: [Kore](https://play.google.com/store/apps/details?id=org.xbmc.kore) use _Po odoslaní ukončiť_ for _Finish after send_.

```suggestion
        "message": "Po odoslaní ukončiť"
```

File: src/_locales/sk/messages.json [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+    },
+    "popup_text_textcontent": {
+        "message": "Zmeniť server:"
+    },
+
+    "options_serverTitle_textcontent": {
+        "message": "Spojenie"
+    },
+    "options_serverIntro_textcontent": {
+        "message": "Aby ste mohli poslať videá/hudbu do Kodi, musíte najprv aplikáciam na iných systémoch povoliť diaľkové ovládanie. Táto voľba je v Kodi v sekcii Nastavenia / Servery / Ovládanie. Potom vyplňte adresu Kodi nižšie. Túto adresu nájdete v sekcii Nastavenia / Systémové informácie / Súhrn v Kodi."
```
Comment (regseb) [inline src/_locales/sk/messages.json:232]: Kodi uses [_Povoliť diaľkové ovládanie pomocou programov v iných systémoch_](https://github.com/xbmc/translations/blob/master/kodi-translations/kodi-main/merged-langfiles/kodi.core/resource.language.sk_sk/resources/strings.po#L2802) for _Allow remote control by programs on other systems_.

```suggestion
        "message": "Aby ste mohli poslať videá/hudbu do Kodi, musíte najprv musíte najprv Povoliť diaľkové ovládanie pomocou programov v iných systémoch. Táto voľba je v Kodi v sekcii Nastavenia / Servery / Ovládanie. Potom vyplňte adresu Kodi nižšie. Túto adresu nájdete v sekcii Nastavenia / Systémové informácie / Súhrn v Kodi."
```

File: locales/sk/amo_description.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      <li>Spojené kráľovstvo: Daily&nbsp;Mail, The&nbsp;Guardian.</li>
+    </ul>
+  </li>
+</ul>
+Cast&nbsp;Kodi tiež analyzuje stránky a hľadá v nich odkazy na videá a hudby z externých platforiem. Ak napríklad stránka obsahuje YouTube video, toto video bude poslané do Kodi.
+
+Pre poslanie videa/hudby môžete do kontextovej ponuky odkazu / stránky / výberu pridať nasledovné 3 položky: <em>Prehrať teraz v Kodi</em>, <em>Prehrať neskôr v Kodi</em> a <em>Pridať do poradia v Kodi</em>. Prítomné je aj diaľkové ovládanie (dostupné na lište s nástrojmi), ktoré vám umožní odoslať aktuálnu kartu do Kodi a pozastaviť video, prejsť na ďalšiu položku, upraviť hlasitosť, ...
+
+
+V Kodi musíte <em>Povoliť diaľkové ovládanie aplikáciam na iných systémoch</em>. Túto možnosť nájdete v <em>Nastavenia</em> / <em>Servery</em> / <em>Ovládanie</em>.
```
Comment (regseb) [inline locales/sk/amo_description.html:26]: Kodi uses [_Povoliť diaľkové ovládanie pomocou programov v iných systémoch_](https://github.com/xbmc/translations/blob/master/kodi-translations/kodi-main/merged-langfiles/kodi.core/resource.language.sk_sk/resources/strings.po#L2802) for _Allow remote control by programs on other systems_.

```suggestion
V Kodi musíte <em>Povoliť diaľkové ovládanie pomocou programov v iných systémoch</em>. Túto možnosť nájdete v <em>Nastavenia</em> / <em>Servery</em> / <em>Ovládanie</em>.
```

**Review by regseb — COMMENTED** [review]
Thank you very much for the translation. I don't speak Slovak, but I did some feedback based on the translations in Kodi.

Can you change the commit message to `feat: Add Slovak translation` ?

File: src/_locales/sk/messages.json [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+        "message": "Poslať do Kodi"
+    },
+    "menus_firstSend": {
+        "message": "Prehrať teraz v Kodi"
+    },
+    "menus_firstInsert": {
+        "message": "Prehrať neskôr v Kodi"
+    },
+    "menus_firstAdd": {
+        "message": "Pridať do poradia v Kodi"
```
[the remark this answers is not in the document]
Comment (jose1711) [reply src/_locales/sk/messages.json:16]: "front" is a colloquial word and very rarely used these in contemporary Slovak. some may refer it to as a "Czechism" (https://en.wikipedia.org/wiki/Bohemism) as it is clearly related to Czech word "fronta". personally I consider "poradie" a better translation.

File: src/_locales/sk/messages.json [hunk 5]
```diff
@@ -0,0 +1,338 @@
...
+    },
+    "menus_secondSend": {
+        "message": "Prehrať teraz"
+    },
+    "menus_secondInsert": {
+        "message": "Prehrať neskôr"
+    },
+    "menus_secondAdd": {
+        "message": "Do poradia"
```
[the remark this answers is not in the document]
Comment (jose1711) [reply src/_locales/sk/messages.json:25]: "front" is a colloquial word and very rarely used these in contemporary Slovak. some may refer it to as a "Czechism" (https://en.wikipedia.org/wiki/Bohemism) as it is clearly related to Czech word "fronta". personally I consider "poradie" a better translation.

File: src/_locales/sk/messages.json [hunk 6]
```diff
@@ -0,0 +1,338 @@
...
+    },
+
+    "popup_send_title": {
+        "message": "Prehrať teraz v Kodi [P]"
+    },
+    "popup_insert_title": {
+        "message": "Prehrať neskôr v Kodi [N]"
+    },
+    "popup_add_title": {
+        "message": "Pridať do poradia v Kodi [Q]"
```
[the remark this answers is not in the document]
Comment (jose1711) [reply src/_locales/sk/messages.json:97]: "front" is a colloquial word and very rarely used these in contemporary Slovak. some may refer it to as a "Czechism" (https://en.wikipedia.org/wiki/Bohemism) as it is clearly related to Czech word "fronta". personally I consider "poradie" a better translation.

File: src/_locales/sk/messages.json [hunk 7]
```diff
@@ -0,0 +1,338 @@
...
+        "message": "Vyberte položky pridané do kontextovej ponuky:"
+    },
+    "options_menuActionsSend_textcontent": {
+        "message": "prehrať teraz"
+    },
+    "options_menuActionsInsert_textcontent": {
+        "message": "prehrať neskôr"
+    },
+    "options_menuActionsAdd_textcontent": {
+        "message": "do poradia"
```
[the remark this answers is not in the document]
Comment (jose1711) [reply src/_locales/sk/messages.json:291]: "front" is a colloquial word and very rarely used these in contemporary Slovak. some may refer it to as a "Czechism" (https://en.wikipedia.org/wiki/Bohemism) as it is clearly related to Czech word "fronta". personally I consider "poradie" a better translation.

File: locales/sk/amo_description.html [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+      <li>Island: Útvarp&nbsp;Saga;</li>
+      <li>Holandsko: Dumpert;</li>
+      <li>Rusko: Первый&nbsp;канал;</li>
+      <li>Spojené kráľovstvo: Daily&nbsp;Mail, The&nbsp;Guardian.</li>
+    </ul>
+  </li>
+</ul>
+Cast&nbsp;Kodi tiež analyzuje stránky a hľadá v nich odkazy na videá a hudby z externých platforiem. Ak napríklad stránka obsahuje YouTube video, toto video bude poslané do Kodi.
+
+Pre poslanie videa/hudby môžete do kontextovej ponuky odkazu / stránky / výberu pridať nasledovné 3 položky: <em>Prehrať teraz v Kodi</em>, <em>Prehrať neskôr v Kodi</em> a <em>Pridať do poradia v Kodi</em>. Prítomné je aj diaľkové ovládanie (dostupné na lište s nástrojmi), ktoré vám umožní odoslať aktuálnu kartu do Kodi a pozastaviť video, prejsť na ďalšiu položku, upraviť hlasitosť, ...
```
[the remark this answers is not in the document]
Comment (jose1711) [reply locales/sk/amo_description.html:23]: "front" is a colloquial word and very rarely used these in contemporary Slovak. some may refer it to as a "Czechism" (https://en.wikipedia.org/wiki/Bohemism) as it is clearly related to Czech word "fronta". personally I consider "poradie" a better translation.

File: src/_locales/sk/messages.json [hunk 9]
```diff
@@ -0,0 +1,338 @@
...
+        "message": "Dole [Arrow Down]"
+    },
+    "popup_osd_title": {
+        "message": "Zobraziť OSD [M]"
+    },
+    "popup_home_title": {
+        "message": "Prejsť domov"
+    },
+    "popup_fullscreen_title": {
+        "message": "Vypnúť/zapnúť celú obrazovku [Tab]"
```
[the remark this answers is not in the document]
Comment (jose1711) [reply src/_locales/sk/messages.json:192]: Equivalents (with Vypnúť/Zapnúť (Toggle) being a little bit more accurate)

File: src/_locales/sk/messages.json (hunk 1 above)
Comment (jose1711) [reply src/_locales/sk/messages.json:213]: Equivalents (order does not change the meaning)

File: src/_locales/sk/messages.json (hunk 2 above)
Comment (jose1711) [reply src/_locales/sk/messages.json:232]: It's pretty much the same. Kodi's translation sounds more like "allow remote control *by using* (with help of) programs on other systems" which I found not too accurate though.

File: locales/sk/amo_description.html (hunk 3 above)
Comment (jose1711) [reply locales/sk/amo_description.html:26]: It's pretty much the same. Kodi's translation sounds more like "allow remote control by using (with help of) programs on other systems" which I found not too accurate though.

**Pull request merged**